examples/routing_service/routing_service_monitoring/c++11/USER_QOS_PROFILES.xml

```diff
@@ -0,0 +1,295 @@
+** Generic.KeepLastReliable.LargeData.FastFlow
+** Generic.KeepLastReliable.LargeData.MediumFlow
+** Generic.KeepLastReliable.LargeData.SlowFlow
+** Generic.KeepLastReliable.TransientLocal
+** Generic.KeepLastReliable.Transient
+** Generic.KeepLastReliable.Persistent
+** Generic.AutoTuning
+** Pattern.PeriodicData
+** Pattern.Streaming
+** Pattern.ReliableStreaming
+** Pattern.Event
+** Pattern.AlarmEvent
+** Pattern.Status
+** Pattern.AlarmStatus
+** Pattern.LastValueCache
+
+Also, following the QoS Profile composition pattern you can use QoS Snippets
+to easily create your final QoS Profile. For further information visit:
+https://community.rti.com/best-practices/qos-profile-inheritance-and-composition-guidance
+
+There is a QoS Snippet library that contains a collection of
+QoS Snippets that set some specific features or configurations.
+
+* Library "BuiltinQosSnippetLib":
+** Optimization.ReliabilityProtocol.Common
+** Optimization.ReliabilityProtocol.KeepAll
+** Optimization.ReliabilityProtocol.KeepLast
+** Optimization.ReliabilityProtocol.HighRate
+** Optimization.ReliabilityProtocol.LowLatency
+** Optimization.ReliabilityProtocol.LargeData
+** Optimization.DataCache.LargeData.DynamicMemAlloc
+** Optimization.Discovery.Common
+** Optimization.Discovery.Participant.Properties.Compact
+** Optimization.Discovery.Endpoint.Fast
+** Optimization.Transport.LargeBuffers
+** QosPolicy.Reliability.Reliable
+** QosPolicy.Reliability.BestEffort
+** QosPolicy.History.KeepLast_1
+** QosPolicy.History.KeepAll
+** QosPolicy.PublishMode.Asynchronous
+** QosPolicy.Durability.TransientLocal
+** QosPolicy.Durability.Transient
+** QosPolicy.Durability.Persistent
+** QosPolicy.Batching.Enable
+** Feature.FlowController.838Mbps
+** Feature.FlowController.209Mbps
+** Feature.FlowController.52Mbps
+** Feature.AutoTuning.Enable
+** Feature.Monitoring.Enable
+** Feature.Security.Enable
+** Feature.TopicQuery.Enable
+** Transport.TCP.LAN.Client
+** Transport.TCP.WAN.Symetric.Client
+** Transport.TCP.WAN.Asymetric.Server
+** Transport.TCP.WAN.Asymetric.Client
+** Compatibility.ConnextMicro.Version243
+** Compatibility.OtherDDSVendor.Enable
+** Compatibility.510Transport.Enable
+
+You should not edit the file BuiltinProfiles.documentationONLY.xml directly.
+However, if you wish to modify any of the values in a built-in profile, the
+recommendation is to create a profile of your own and inherit from the built-in
+profile you wish to modify. The NDDS_QOS_PROFILES.example.xml file (contained in 
+the same directory as the BuiltinProfiles.documentationONLY.xml file) shows how
+to inherit from the built-in profiles. 
+
+For more information about XML QoS Profiles see the "Configuring QoS with
+XML" chapter in the RTI Connext DDS Core Libraries User's Manual.
+-->
+
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:noNamespaceSchemaLocation="/Applications/rti_connext_dds-6.0.1/resource/schema/rti_dds_qos_profiles.xsd">
```

Comment:
Replace with the following URL: `http://community.rti.com/schema/6.1.0/rti_dds_qos_profiles.xsd`

Reply:
Done!